plugins/grid-bookmark/src/GridBookmarkWidget/components/BookmarkGrid.tsx

```diff
@@ -59,10 +61,8 @@ const BookmarkGrid = observer(function ({
       }
     })
 
-  // reset selections if bookmarked regions change
-  // needed especially if bookmarked regions are deleted, then
   const [widths, setWidths] = useState([
-    50,
+    40,
```

Comment:
I think that the checkbox cell is still width ~50 so this change may not be accurate. can see in hovering the lines are mismatched

![image](https://github.com/GMOD/jbrowse-components/assets/6511937/392ae9ec-a407-4eda-8c8c-04385e2c99aa)